cron/daily/make-private-updates-public.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+import models, { Op } from '../../server/models';
+import logger from '../../server/lib/logger';
+
+// get the ISOtime for the day @ midnight to compare
+// against the update.makePublic on time field
+
+const today = new Date(new Date().setUTCHours(0, 0, 0, 0)).toISOString();
+logger.info(`Updates for ${today}`);
+
+// get all the private updates with the makePublicOn <= today
+// and make them public
+
+models.Update.update(
```

Comment:
I would extract that to the model to be easily available elsewhere, ie: `models.Update.makePrivateUpdatesPublic`

Reply:
extracted to a new Update method. (.makePublic())